server/py/services/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -148,25 +144,18 @@ def create_model_endpoint(
                 feature_names=model_endpoint.spec.feature_names,
             )
 
-        # If none of the above was supplied, feature names will be assigned on first contact with the model monitoring
-        # system
-        logger.info("Creating model endpoint", endpoint_id=model_endpoint.metadata.uid)
-        # Write the new model endpoint
-        model_endpoint_store = (
-            services.api.crud.model_monitoring.helpers.get_store_object(
-                project=model_endpoint.metadata.project
-            )
+        # 5. write the model endpoint to the db again
+        framework.utils.singletons.db.get_db().update_model_endpoint(
+            session=db_session,
+            project=model_endpoint.metadata.project,
+            name=model_endpoint.metadata.name,
+            function_name=model_endpoint.spec.function_name,
+            attributes=attributes,
+            uid=model_endpoint.metadata.uid,
         )
-        model_endpoint_store.write_model_endpoint(endpoint=model_endpoint.flat_dict())
 
-        if (
-            model_endpoint.spec.monitoring_mode
-            == mlrun.common.schemas.model_monitoring.ModelMonitoringMode.enabled
-        ):
-            # Create model monitoring schedules file
-            ModelMonitoringSchedulesFile.from_model_endpoint(model_endpoint).create()
-            # Create model monitoring stats files:
-            cls._create_model_monitoring_stats_files(model_endpoint=model_endpoint)
```

Comment:
I put here deliberately here so that one can use the MM framework even without `model_endpoint.spec.model_uri`.